src/components/SettingsBtn.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useRef } from 'react';
```

Comment:
same as above

Reply:
So it fixed

Reply:
it is not fixed, file contains indentation 4 spaces, should be 2